.travis.yml

```diff
@@ -1,7 +1,14 @@
-language: rust
-rust:
-  - stable
+language: node_js
+node_js:
+  - 12.0
+os:
+  - linux
```

Comment:
We run `npm run build` in this script and it wasn't finding it with the language being Rust. I found the "recipe" for this online where it downloads Rust and has npm available. Weird, but Travis is weird, and having multiple languages cross-platform is also weird. (Windows uses `nvs` instead of `nvm`, etc…) I looked into this for a while and found this to be the solution.